mysqld_exporter.go

```diff
@@ -1643,6 +1666,33 @@ func scrapeTableSchema(db *sql.DB, ch chan<- prometheus.Metric) error {
 	}
 
 	for _, database := range dbList {
+		if _, ok := tableSchemaLastRun[database]; !ok {
```

Comment:
You can actually skip the whole thing.
A map will return you the zero value if the element doesn't exist. Thus `tableSchemaLastRun["neverseenbefore"] == time.Time{}`.